src/App.css

```diff
@@ -0,0 +1,39 @@
+.App {
```

Comment:
- [ ] In this screen in the mobile view, kindly touch up your styling so the content fits the screen and appears more centered:

![image](https://github.com/jonas-45/motorcycle-booking-react/assets/57421823/054f11f9-c0e5-429a-8180-dcf9e3acd880)